recipes/quaternionic/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set owner = "moble" %}
+{% set name = "quaternionic" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ owner }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: c83bd808174621c38b96a103b14a772733cb70e874800170ae15bb132b439019
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
```

Comment:
This can be a noarch: python package

Reply:
Sorry.  I always forget how to make it noarch but still limit the python versions.